src/app/renderer/actions/fileTree/selectFilter.ts

```diff
@@ -0,0 +1,8 @@
+import {SelectFilter} from '../../../../domain/useCases/fileTree/SelectFilter';
+import {container} from '../../container';
+
+const useCase: SelectFilter = container.get(SelectFilter);
+
+export function selectFilter(filter: string) {
```

Comment:
should have return type `: Promise<void>`

Reply:
done